SpatialGDK/Source/SpatialGDKEditor/Private/SchemaGenerator/SpatialGDKEditorSchemaGenerator.cpp

```diff
@@ -29,11 +29,13 @@
 #include "Utils/DataTypeUtilities.h"
 #include "Utils/SchemaDatabase.h"
 #include "Engine/WorldComposition.h"
+#include "Misc/ScopedSlowTask.h"
+#include "UObject/StrongObjectPtr.h"
 
 DEFINE_LOG_CATEGORY(LogSpatialGDKSchemaGenerator);
+#define LOCTEXT_NAMESPACE "SpatialGDKSchemaGenerator"
```

Comment:
What is this for? Just for my  understanding.

Reply:
The namespace must be defined in order to use the LOCTEXT macro.